src/acquisition/ght/ght_update.py

```diff
@@ -305,7 +306,12 @@ def get_num_rows():
         ew = result['start_week']
         num_missing = 0
         for v in values:
-          sql_data = (term, location, ew, v, v)
+          sql_location = location
+          if country != 'US':
+            sql_location = country + "_"
```

Comment:
Are you sure you want to add the underscore here? If there is no location (i.e. country code only), then if I'm reading it correctly it'll end up like "MX_" for example. Do you want to add the underscore inside the `if` below so that it's only used when there's a location?

Reply:
The underscore is meant to distinguish between countries and states, i.e. CA can mean both Canada and California. I added a comment to clarify. 
In the future, maybe we can store countries based on the ISO 1366 alpha-3 code (3 letters for the country), but that would require some look-up table to translate between alpha-2 and alpha-3 codes.

Reply:
Ok, makes sense.